shared/message/motion/Limbs.proto

```diff
@@ -0,0 +1,83 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2022 NUbots <nubots@nubots.net>
+ */
+syntax = "proto3";
+
+package message.motion;
+
+import "message/motion/ServoCommand.proto";
+
+/// Left leg servo command information including time, gain, torque, position
+message LeftLeg {
+    enum ID {
+        LEFT_HIP_YAW     = 0;
+        LEFT_HIP_ROLL    = 1;
+        LEFT_HIP_PITCH   = 2;
+        LEFT_KNEE        = 3;
+        LEFT_ANKLE_PITCH = 4;
+        LEFT_ANKLE_ROLL  = 5;
+    }
+    /// Information for each servo in the left leg, in the order of ID
+    repeated ServoCommand servos = 1;
+}
+
+/// Right leg servo command information including time, gain, torque, position
+message RightLeg {
+    enum ID {
+        RIGHT_HIP_YAW     = 0;
+        RIGHT_HIP_ROLL    = 1;
+        RIGHT_HIP_PITCH   = 2;
+        RIGHT_KNEE        = 3;
+        RIGHT_ANKLE_PITCH = 4;
+        RIGHT_ANKLE_ROLL  = 5;
+    }
+    /// Information for each servo in the right leg, in the order of ID
+    repeated ServoCommand servos = 1;
```

Comment:
Not sure, but it might be worth making this a map rather than a vector.
Is there a situation where you wouldn't want to set all of them?